code/game/objects/structures/crates_lockers/largecrate_vr.dm

```diff
@@ -35,4 +35,41 @@
 							 "<span class='notice'>You hear splitting wood.</span>")
 		qdel(src)
 	else
-		return attack_hand(user)
\ No newline at end of file
+		return attack_hand(user)
+
+/obj/structure/largecrate/animal/pred
+	name = "Predator carrier"
+	held_type = /mob/living/simple_animal/catgirl
+
+/obj/structure/largecrate/animal/pred/New() //This is nessesary to get a random one each time.
+
+	held_type = pick(/mob/living/simple_animal/hostile/bee,
+						/mob/living/simple_animal/catgirl,
```

Comment:
If you're going to include 3 of the same thing within a pick list, you'd be as well making it weighted so you don't need 3 of the same entry.